src/js/components/aboutTheData/AboutTheDataPage.jsx

```diff
@@ -67,23 +68,23 @@ const AboutTheDataPage = ({
     };
 
     useEffect(() => {
-        if ((!urlFy || !urlPeriod) && submissionPeriods.size && latestFy && latestPeriod) {
+        if ((!urlFy || !urlPeriod || !activeTab) && submissionPeriods.size && latestFy && latestPeriod) {
```

Comment:
This might be out of scope but if a user does not provide a fiscal year or period or activeTab we send them to the submissions tab by default? What if a user wants to visit `submission-statistics/?tab=publications`?

Reply:
This is addressed in the tests. If they provide any variation of an incomplete query param, we support it w/ defaults; ie, the submissions tab, the latest fy, and the latest period.